app/templates/components/confirmation-modal.hbs

```diff
@@ -0,0 +1,5 @@
+<div id="hearing-confirmation" class="reveal-overlay" style="display:block;">
```

Comment:
The `id` should be more generic if this is going to be used in places other then the hearing form.

Reply:
Actually, IDK if an `id` is even necessary.